torchci/lib/bot/triggerInductorTestsBot.ts

```diff
@@ -0,0 +1,66 @@
+import { Context, Probot } from "probot";
+
+export default function triggerInductorTestsBot(app: Probot): void {
+  const preapprovedUsers = ["pytorchbot", "PaliC"]; // List of preapproved users
+  //   const tritonRepo = "triton-lang/triton"; // uncomment once this is good enough for triton
+  const tritonRepo = "triton-lang-test/triton"; // delete once this is good enough for triton
+  const preapprovedRepos = ["malfet/deleteme", tritonRepo]; // List of preapproved orgs/repos
+  //   const preapprovedRepos = ["malfet/deleteme", "triton-lang/triton"];// uncomment once this is good enough for triton and delete line above
+
+  app.on(
+    ["issue_comment.created"],
+    async (ctx: Context<"issue_comment.created">) => {
+      const commentBody = ctx.payload.comment.body.toLowerCase();
+      const commenter = ctx.payload.comment.user.login;
+      const orgRepo = `${ctx.payload.repository.owner.login}/${ctx.payload.repository.name}`;
+      if (
+        commentBody.includes("trigger inductor tests") &&
+        preapprovedUsers.includes(commenter) &&
+        preapprovedRepos.includes(orgRepo)
+      ) {
+        const workflow_owner = "pytorch";
+        const workflow_repo = "pytorch-integration-testing";
+        const workflow_id = "triton-inductor.yml";
+
+        let tritonCommit = "main";
+        // if on the triton repo, get the commit of the pr
+        if (orgRepo === tritonRepo) {
+          const pr = await ctx.octokit.pulls.get({
+            owner: ctx.payload.repository.owner.login,
+            repo: ctx.payload.repository.name,
+            pull_number: ctx.payload.issue.number,
+          });
+          tritonCommit = pr.data.head.sha;
+        }
+
+        try {
+          await ctx.octokit.actions.createWorkflowDispatch({
+            owner: workflow_owner,
+            repo: workflow_repo,
+            workflow_id,
+            ref: "main",
+            inputs: {
+              triton_commit: tritonCommit,
+              pytorch_commit: "viable/strict",
```

Comment:
A thought, what happen if we make `pytorch_commit` a parameter, i.e. `@pytorchbot trigger inductor test on pytorch main` or `@pytorchbot trigger inductor test on pytorch SHA`.  The default can be `viable/strict`.  That would be nice